DittoHeartbeat/src/main/java/live.ditto.dittoheartbeat/Heartbeat.kt

```diff
@@ -94,7 +94,7 @@ fun addToCollection(info: DittoHeartbeatInfo, config: DittoHeartbeatConfig, ditt
         "metaData" to metaData,
         "sdk" to info.sdk,
         "_schema" to info.schema,
-        "peerKey" to info.peerKey
+        "peerKeyString" to info.peerKeyString
```

Comment:
I think the key name here actually needs to stay the same, Swift still has it as `peerKey` as far as I can tell.